static/js/tools/map/time_slider.tsx

```diff
@@ -161,6 +164,27 @@ export function TimeSlider(props: TimeSliderProps): JSX.Element {
                 x1={SLIDER_MARGIN}
                 x2="100%"
               ></line>
+              {document.getElementById("time-slider-slide") &&
+                ticks.map((element) => {
```

Comment:
nit: "element" is a bit of a confusing name for the variable because each tick isn't yet an element .. maybe just "tick" or "date"